api/src/main/java/io/grpc/InternalManagedChannelBuilder.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc;
+
+/**
+ * Internal accessors for {@link ManagedChannelBuilder}.
+ */
+public final class InternalManagedChannelBuilder {
```

Comment:
What am I missing here - what is this needed for? I don't see it being used anywhere.

Reply:
See OpenTelemetryModule and ManagedChannelImplBuilderTest.

Reply:
I think I saw those, then later went back to take another look and just couldn't find them. Sigh.

Reply:
In your defense, if you were reviewing each commit separately you wouldn't have seen the usage in OpenTelemetryModule. :shrug: